### PR TITLE
Add automatic tests in Appveyor

### PR DESCRIPTION
**Pull request by barbelity**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+stack: jdk 8
+environment:
+  matrix:
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
```
Comment (yahavi) [inline appveyor.yml]: It is better to use images like here:
https://github.com/jfrog/jfrog-idea-plugin/blob/master/appveyor.yml#L1

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+stack: jdk 8
```
Comment (yahavi) [inline appveyor.yml]: Please add also `skip_tags: true`

File: appveyor.yml (hunk 1 above)
Comment (barbelity) [reply appveyor.yml]: We will leave it that way to avoid setting a redundant env_var on the linux machine

File: appveyor.yml (hunk 2 above)
Comment (barbelity) [reply appveyor.yml]: done

**Pull request merged**